node/sync.go

```diff
@@ -202,6 +204,11 @@ func (p *syncProviderWrapper) updatePodStatus(ctx context.Context, podFromKubern
 		log.G(ctx).Debug("Setting pod not found on pod status")
 	}
 
+	if podStatus == nil {
+		log.G(ctx).WithError(statusErr).Debug("Skipping pod status update due to missing pod status")
```

Comment:
This seems like a higher-level message than debug?

Reply:
Sure, I updated this to actually return the error.